src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +55,45 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService =
+                (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        UserService userService =
+                (UserService) injector.getInstance(UserService.class);
+        AuthenticationService authenticationService
+                = (AuthenticationService) injector.getInstance(AuthenticationService.class);
+        User testUserOne = null;
+        User testUserTwo = null;
+        try {
+            authenticationService.register("newuserOne@gmail.com", "qwe123");
+            authenticationService.register("newuserTwo@gmail.com", "123qwe");
+        } catch (Exception e) {
+            throw new RuntimeException("Can't register user.");
+        }
+        try {
+            testUserOne = authenticationService.login("newuserOne@gmail.com", "qwe123");
+            testUserTwo = authenticationService.login("newuserTwo@gmail.com", "123qwe");
+        } catch (AuthenticationException e) {
+            throw new RuntimeException("Can't login with given properties.");
+        }
```

Comment:
I think, in this project there is no point in this try catch blocks, you can just register new User

Reply:
If try catch blocks will be removed, methods login() an register() may throw an Exception, so in terms of good practice we should catch exceptions to prevent adding `throws SomeException`  in main signature